open_academy/models/open_academy_course.py

```diff
@@ -10,3 +10,21 @@ class OpenAcademyCourse(models.Model):
 
     responsible_id = fields.Many2one('res.users')
     session_ids = fields.One2many('open.academy.session', 'course_id')
+
+    _sql_constraints = [
+        ('name_description',
+         'CHECK(name != description)',
+         "The course title must be different to the description"),
+
+        ('name_unique',
+         'UNIQUE(name)',
+         "The course title must be unique"),
+    ]
+
+    @api.returns("self", lambda value: value.id)
+    def copy(self, default=None):
+        if default is None:
+            default = {}
+        if not default.get("name"):
+            default["name"] = "Copy of " + self.name
+        return super(OpenAcademyCourse, self).copy(default)
```

Comment:
Did you use precommit on this change?

Call the super method with parameters is no longer used, please run precommit and change this.